data.js

```diff
@@ -5922,5 +5922,13 @@ module.exports = [
     description: "A small entity-component-system library written in JS",
     url: "https://github.com/Stuhl/javascript-entity-component-system",
     source: "https://raw.githubusercontent.com/Stuhl/javascript-entity-component-system/master/src/index.js"
-  }
+  },
+  {
+    name: "spect",
+    github: "spectjs/spect",
+    tags: ["aspect", "aop", "frp", "observable", "selector-observer", "hyperscript", "htm"],
+    description: "Aspect-oriented reactive DOM framework",
+    url: "https://github.com/spectjs/spect",
+    source: "https://unpkg.com/browse/spect@22.0.0/dist/spect.min.js"
```

Comment:
This should be a link to raw and unminified source code. Please refer to the repository's readme, which lays out what is expected of submitted packages.

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified

Reply:
Sorry for the delay. I've been pretty absorbed with other things. Unfortunately I don't think there's anyone very actively maintaining this repository at the moment.